webapp/channels/src/components/purchase_modal/delinquency_card.tsx

```diff
@@ -0,0 +1,127 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {FormattedMessage} from 'react-intl';
+
+import {trackEvent} from 'actions/telemetry_actions';
+
+import {TELEMETRY_CATEGORIES, CloudLinks} from 'utils/constants';
+
+import type {ButtonDetails} from './purchase_modal';
+
+type DelinquencyCardProps = {
+    topColor: string;
+    price: string;
+    buttonDetails: ButtonDetails;
+    onViewBreakdownClick: () => void;
+    isCloudDelinquencyGreaterThan90Days: boolean;
+    users: number;
+    cost: number;
+};
+
+export default function DelinquencyCard(props: DelinquencyCardProps) {
```

Comment:
This was all existing code, just moved it out of the purchase_modal.tsx file because it was getting a little narly